README.md

```diff
@@ -81,10 +81,13 @@ If you do not have those please consider getting a [docking station](https://eu.
 ### Setup Raspberry Pi 5
 
 1. Prepare your Raspberry Pi 5 device to be ready to start according to official
-[getting started](https://www.raspberrypi.com/documentation/computers/getting-started.html) manual;
+[getting started](https://www.raspberrypi.com/documentation/computers/getting-started.html) manual. It is not necessary
+to install an operating system as described in the above document as AosEdge image will be installed further;
```

Comment:
See comment in the PR I think we shall avoid referring the "Getting started" document and instead say something like: "Connect power supply and insert micro-SD card"